app/controllers/api/v1/rents_controller.rb

```diff
@@ -0,0 +1,28 @@
+module Api
+  module V1
+    class RentsController < ApiController
+      include Wor::Paginate
+      def index
+        rents = Rent.where(['user_id = :uid', { uid: params[:user_id] }])
+        render_paginated rents, each_serializer: RentIndexSerializer
+      end
+
+      def create
+        if current_user.id != params[:rent][:user_id]
+          render json: { message: 'User id not allowed' }, status:	:unprocessable_entity
+          return
+        end
+        rent = Rent.new(rent_params)
+        if rent.save
+          render json: rent, serializer: RentIndexSerializer, status: :created
+        else
+          render json: { message: 'Bad request' }, status:	:bad_request
```

Comment:
Here the 4xx status is okay :ok_hand: , but it should be `unprocessable_entity` instead of `bad_request`, check out this link: https://softwareengineering.stackexchange.com/questions/329229/should-i-return-an-http-400-bad-request-status-if-a-parameter-is-syntactically/329233

Also, we can return the errors so the client who did this request knows what to fix, using something like: 

`render json: rent.errors, status: :unprocessable_entity`